### PR TITLE
Contribute/Create listing updated UI component

### PR DESCRIPTION
**Pull request by pfeeny5**
# Pull Request Template

## Issue

Addresses # (issue)

- [x] This change addresses the issue in full
- [ ] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description
Created a new panel for the listing sidebar that includes the listing updated date. This change creates the UI-component and implements it on the listing page.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Prototype/POC (not to merge)
- [ ] This change is a refactor/address technical debt
- [ ] This change requires a documentation update
- [ ] This change requires a SQL Script

## How Can This Be Tested/Reviewed?
Add a <ListingUpdated> component to any page.

- [x] Desktop View
- [x] Mobile View
- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have reviewed the changes in a desktop view
- [ ] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have updated the changelog to include a description of my changes

### CODE REVIEW

File: ui-components/src/page_components/listing/listing_sidebar/ListingUpdated.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import * as React from "react"
+import { t } from "../../../helpers/translator"
+import { Listing } from "@bloom-housing/backend-core/types"
+import moment from "moment"
+
+interface ListingUpdatedProps {
+  listing: Listing
```
Comment (emilyjablonski) [inline ui-components/src/page_components/listing/listing_sidebar/ListingUpdated.tsx:7]: We're working to remove dependencies on the backend from ui-components, so we don't want to be adding any new ones. Can the content here just be passed in through the prop and can we remove `Listing` ?

File: ui-components/src/page_components/listing/listing_sidebar/ListingUpdated.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+const ListingUpdated = (props: ListingUpdatedProps) => {
+  const listing = props.listing
+  return (
+    <section className="aside-block">
+      <p className="text-tiny text-gray-800">
+        {t("listings.listingUpdated")}
```
Comment (emilyjablonski) [inline ui-components/src/page_components/listing/listing_sidebar/ListingUpdated.tsx:15]: Would prefer to see template literals here for readability (\`${t("listings.listingUpdated")}: ${moment...}\`)

File: ui-components/src/page_components/listing/listing_sidebar/ListingUpdated.tsx (hunk 1 above)
Comment (pfeeny5) [reply ui-components/src/page_components/listing/listing_sidebar/ListingUpdated.tsx]: Updated to just pass the updated through as a prop vs the entire Listing.

File: ui-components/src/page_components/listing/listing_sidebar/ListingUpdated.tsx (hunk 2 above)
Comment (pfeeny5) [reply ui-components/src/page_components/listing/listing_sidebar/ListingUpdated.tsx]: Thanks for the suggestion.  Updated here in latest commit.

**Comment by seanmalbert** [comment]
@emilyjablonski , can you review this with the updates and merge if it looks good?

**Review by emilyjablonski — APPROVED** [review]

**Pull request merged**